404.html

```diff
@@ -11,60 +11,177 @@
+      }
+
+      @media (min-width: 80rem) {
+        font-size: var(--spectrum-font-size-1400);
+      }
+
+      @media (min-width: 105rem) {
+        font-size: var(--spectrum-font-size-1500);
+      }
     }
 
-    main.error .error-number {
-      width: 100%;
+    .not-found__heading--careers {
+      margin-block-end: var(--spacing-global-block-end-small-screen);
+
+      @media (min-width: 48rem) {
+        margin-block-end: 8rem;
+      }
+
+      @media (min-width: 80rem) {
+        margin-block-end: 12rem;
+      }
     }
 
-    main.error .error-number {
-      fill:  var(--color-text);
+    .not-found__description,
+    .not-found__image {
+      @media (min-width: 48rem) {
+        max-width: 25.5rem;
+      }
+
+      @media (min-width: 80rem) {
+        max-width: 33rem;
+      }
+
+      @media (min-width: 105rem) {
+        max-width: 34.5rem;
+      }
+    }
+
+    .not-found__description {
+      font-family: var(--spectrum-sans-serif-font);
+      font-weight: var(--spectrum-bold-font-weight);
+      font-size: var(--spectrum-font-size-300);
+      line-height: 1.3;
+      margin-block-start: 0;
+      margin-block-end: 1rem;
+
+      @media (min-width: 48rem) {
+        font-size: var(--spectrum-font-size-500);
+        margin-block-end: 2rem;
+      }
+
+      @media (min-width: 80rem) {
+        font-size: var(--spectrum-font-size-700);
+        font-weight: var(--spectrum-extra-bold-font-weight);
+        margin-block-end: 3rem;
+      }
+
+      @media (min-width: 105rem) {
+        font-size: var(--spectrum-font-size-750);
+      }
     }
 
-    main.error .error-number text {
-      font-family: var(--fixed-font-family);
+    .not-found__image {
+      margin-inline: 0;
+      margin-block-start: 0;
+      margin-block-end: 2rem;
+
+      @media (min-width: 48rem) {
+        margin-block-end: 3rem;
+      }
+
+      @media (min-width: 80rem) {
+        margin-block-end: 4rem;
+      }
+    }
+
+    .not-found__image figcaption {
+      margin-block-start: 0.25rem;
+
+      @media (min-width: 48rem) {
+        margin-block-start: 0.5rem;
+      }
+
+      @media (min-width: 80rem) {
+        margin-block-start: 1rem;
+      }
     }
   </style>
 </head>
 
 <body>
-  <header></header>
-  <main id="main-content" class="error">
-    <div class="section">
-      <svg viewBox="1 0 38 18" class="error-number">
-        <text x="0" y="17">404</text>
-      </svg>
-      <h2 class="error-message">Page Not Found</h2>
-      <p class="button-container">
-        <a href="/" class="button secondary error-button-home">Go home</a>
-      </p>
-    </div>
+  <header>
+    <!-- Necessary to load nav -->
+  </header>
+  <main id="main-content">
+    <section class="not-found section">
+      <h1 class="not-found__heading">Oops!</h1>
+      <p class="not-found__description">We couldn't find what you're looking for, but we did find this in our design archives:</p>
+      <figure class="not-found__image">
```

Comment:
In case fetching doesn't work, there's at least a default 404 image/description included.